rosidl_generator_rs/package.xml

```diff
@@ -13,7 +13,8 @@
   <buildtool_depend>rosidl_runtime_rs</buildtool_depend>
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
-  <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
+  <buildtool_export_depend condition="$ROS_DISTRO != rolling">rosidl_cmake</buildtool_export_depend>
```

Comment:
I may be interpreting [REP 149](https://www.ros.org/reps/rep-0149.html#id20) incorrectly, but to me it reads as if I should be able to use a less/greater than for ROS distros. Unfortunately that doesn't work, and the xml fails to parse. Not sure what to do if rosidl_cmake 3.3.0+ gets back ported to Humble or when Iron comes out

Reply:
Reads that way to me too. Did you try only `>` or also `>=`?

Reply:
Coming back to this now, looks to only be a problem with less than operators.

Fails to parse 
`$ROS_DISTRO <= rolling`
`$ROS_DISTRO < rolling`

Successfully parses 
`$ROS_DISTRO >= rolling`
`$ROS_DISTRO > rolling`

Thankfully this means we can handle this a bit more concisely in the `package.xml`

```xml
<buildtool_export_depend condition="rolling > $ROS_DISTRO">rosidl_cmake</buildtool_export_depend>
<buildtool_export_depend condition="$ROS_DISTRO >= rolling">rosidl_pycommon</buildtool_export_depend>
```
And then we can swap the usage of `rolling` to `iron` once its out.